tracks/julia/exercises/clock/mentoring.md

```diff
@@ -0,0 +1,118 @@
+## Things to watch out for
+
+- It is preferred to use \``value(m)`\` rather than \``m.value`\`. Fields are generally considered private in Julia. You can import \``value`\` from Dates.
```

Comment:
I think an explanation of why and when this is the case would be helpful. On first look, this sounds like one has to add Java levels of boilerplate without (m)any benefits.

Reply:
That's fair, but I'm not sure I have a good explanation beyond the same reasons that you'd do it for Java or any other language. Can you think of one?

Reply:
In some cases it makes the code more generic. `value(x)` will work for any x as long as there's a method for it, while `x.value` will require the field. You won't reap the benefits of multiple dispatch if you always access fields directly.

Reply:
I feel like that's the same benefit as in Java :shrug: But we could restate it, I suppose for the scientists and novices who might not be familiar.